eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java

```diff
@@ -152,6 +115,29 @@ public void exec() throws IOException {
         }
     }
 
+    /**
+     * Get objectionary by given hash from the map.
+     * @param hash Hash.
+     * @return Objectionary by given hash.
+     */
+    private Objectionary objectionaryBy(final CommitHash hash) {
+        return this.putIfAbsent(
```

Comment:
@maxonfjvipon This code is misleading and it is the main drawback of the `inheritance`. Why `Mojo` has `putIfAbsent` method? Is `Mojo` a map? I don't think so.